config/ConfigParse.cpp

```diff
@@ -99,24 +104,58 @@ std::vector<std::string>	parseMethod(const std::vector<std::pair<std::string, in
 	while (iss >> method)
 		methods.push_back(method);
 	return methods;
+}*/
```

Comment:
Cool approach to contain multiple information about not only the methods in integer, using bit-operator. Could you comment/connect the code how did you use it?